solutions/devsprint-eduardo-bocato-2/FinanceAppTests/Mock/HomeViewModelDelegateMock.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  HomeViewModelDelegateMock.swift
+//  FinanceAppTests
+//
+//  Created by Murillo R. Araújo on 03/05/22.
+//
+
+import Foundation
+@testable import FinanceApp
+
+class HomeViewModelDelegateMock: HomeViewModelDelegate {
+
+    var displayedHomeData = false
+    var displayedError = false
+
+    func didFetchHomeData(_ data: HomeData) {
+        displayedHomeData = true
+    }
+}
```

Comment:
**[CHANGE REQUEST]**
Please use a spy and configure the scenarios in your test functions.
```suggestion
final class HomeViewModelDelegateSpy: HomeViewModelDelegate {
    private(set) var didFetchHomeDataCalled = false
    private(set) var homeDataPassed: HomeData?

    func didFetchHomeData(_ data: HomeData) {
        didFetchHomeDataCalled = true
        homeDataPassed = data
    }
}
```

Reply:
Done